### PR TITLE
alphabetized & expanded the hash words for Prime 1

### PR DESCRIPTION
**Pull request by Shaleblade**
I prioritized words that came up a lot in room/enemy names, as well as a few key Chozo Lore terms.

### CODE REVIEW

**Comment by ThanatosGit** [comment]
Your JSON is invalid It has several occurences of the `,` being before `"` like in `"Crash,"`

**Comment by Shaleblade** [comment]
Apologies - I think it should be fixed now

**Comment by ThanatosGit** [comment]
> Apologies - I think it should be fixed now

Eventually, I don't like approving something for a game, I never touched.
But I can tell you that the bare minimum will be to pass the tests 😉.
Without looking at it, they are probably failing because the hash words are in some test files like rdvgame and tests for the patch data factory.

File: randovania/games/prime1/assets/hash_words.json [hunk 1]
```diff
@@ -1,99 +1,138 @@
...
-    "Super",
-    "Charge",
-    "Wavebuster",
-    "Spreader",
-    "Core",
-    "Flamethrower"
-]
\ No newline at end of file
+ 	"Access",
+	"Aether",
```
Comment (Miepee) [inline randovania/games/prime1/assets/hash_words.json:3]: IDK if it's just me, but I associate Aether waaay more with echoes than prime 1. Where is that even used there?

**Review by Miepee — CHANGES_REQUESTED** [review]
Missing changelog entry

File: randovania/games/prime1/assets/hash_words.json (hunk 1 above)
Comment (Shaleblade) [reply randovania/games/prime1/assets/hash_words.json:3]: IIRC, it was for Research Lab Aether. I'm happy to remove or replace it, though - it's fair that it's way more associated with Echoes.

**Review by toasterparty — APPROVED** [review]

**Comment by toasterparty** [comment]
pending the changelog entry of course

**Pull request merged**